recipe-client-addon/test/browser/browser_NormandyDriver.js

```diff
@@ -12,6 +12,16 @@ add_task(withDriver(Assert, async function uuids(driver) {
   isnot(uuid1, uuid2, "uuids are unique");
 }));
 
+add_task(withDriver(Assert, async function installXpi(driver) {
+  // Test that we can install an XPI from any URL
+  const dir = getChromeDir(getResolvedURI(gTestPath));
+  dir.append("fixtures");
+  dir.append("normandy.xpi");
+  const xpiUrl = Services.io.newFileURI(dir).spec;
+  var addonId = await driver.installAddon(xpiUrl);
+  await driver.uninstallAddon(addonId);
```

Comment:
I think we should actually just do a quick check to see that the add-on ID we expect is in the add-on list, and then isn't after uninstall, just to be thorough. If that's difficult, then at least a comment explaining the return values would help make this test more readable.